### PR TITLE
Game: Adjusted damage, nameplate tracing and weapon switch shooting delay

### PR DESCRIPTION
**Pull request by Alf21**
This should make the game more "enjoyable" and prevent a scroll-lock shooting bug

### CODE REVIEW

File: code/weapons/Pistol.cs [hunk 1]
```diff
@@ -29,6 +29,11 @@ public override bool CanPrimaryAttack()
 
         public override void AttackPrimary()
         {
+            if (TimeSinceDeployed < 0.4f)
```
Comment (mzegar) [inline code/weapons/Pistol.cs:32]: Is there any way to use the `deploy` animation instead of just hard coding values?

**Review by mzegar — COMMENTED** [review]
Comment

File: code/weapons/Pistol.cs (hunk 1 above)
Comment (Alf21) [reply code/weapons/Pistol.cs:32]: Yea, for sure thought the same before searching these values, but didn't found the right method / var if there was one. Gonna try to add a var to Weapons.cs so we don't need to check it in every weapon manually (maybe found a better way).

**Review by mzegar — APPROVED** [review]
LGTM

**Pull request merged**